KlaviyoSwiftExtension.podspec

```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |s|
+  s.name             = "KlaviyoSwiftExtension"
```

Comment:
This is the new pod that is now exported exclusively for the notification service extension. Refer PR description for more info.